src/main/java/codeu/model/store/basic/MentionStore.java

```diff
@@ -67,6 +73,43 @@ public Mention getMentionById(UUID id) {
     }
     return null;
   }
+  
+  /**
+   * Access the current list of mentions by the given user UUID.
+   *
+   * @return null if the user has not mentioned another user.
+   */
+  public List<Mention> getMentionsByUserId(UUID id) {
+	  List<Mention> result = new ArrayList<>();
+	  for (Mention mention : mentions) {
+		  if (mention.getUserWhoDidTheMentioning().equals(id)) {
+			  result.add(mention);
+		  }
+	  }
+	  if (result.size() > 0) {
+		  return result;
+	  }
+	  return null;	  
```

Comment:
return an empty list instead, and you'll never have to say you're sorr...check for null in a caller.